integration-tests/modules/__tests__/cart/store/carts.spec.ts

```diff
@@ -88,6 +90,8 @@ medusaIntegrationTestRunner({
           name: "Default Region",
           currency_code: "dkk",
         })
+
+        await createDefaultsWorkflow(appContainer).run()
```

Comment:
Wouldn't this have already run at this point in time?

Reply:
Only on the first test, we truncate the database after each test

Reply:
aaaah yes, you're right 👍 that might be a bit unexpected I'd say, I'd probably imagine each test running in the exact same environment. But anyway nothing to action, maybe just leave a comment?

Reply:
Jumping Quickly, but maybe this should be part of the runner? Wdyt?

Reply:
That was my first thought, I didn't know if the plans for the runner is to be core-specific. Shall we throw it in there?

Reply:
I believe that the runner already run the main medusa loader and i thought it was already part if it. If you can check that in the runner and maybe we can validate that part, ill be back tomorrow

Reply:
no rush :) I'll take a look